src/main/java/frc/team2412/robot/commands/intake/IntakeBitmapCommand.java

```diff
@@ -19,24 +21,24 @@
     public enum Bitmap {
         // ingesthasball, feederhasball, ingestcorrectcolor, feedercorrectcolor, intakespeed, ingestspeed,
         // feederspeed, misfire
-        A(false, false, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, false), // no balls in systems
-        B(true, false, true, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, false), // correct ball in ingest
-        C(true, false, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, true), // wrong ball in ingest
-        D(false, true, false, true, INTAKE_IN_SPEED, INDEX_IN_SPEED, 0, false), // correct ball in feeder
-        E(false, true, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, true), // wrong ball in feeder
-        F(true, true, true, true, 0, 0, 0, false), // correct ball in both
-        G(true, true, false, true, INTAKE_OUT_SPEED, INDEX_OUT_SPEED, 0, false), // wrong ball in ingest, corect ball in
-                                                                                    // shooter (may change later)
-        H(true, true, true, false, 0, INDEX_IN_SPEED, INDEX_IN_SPEED, true), // correct ball in ingest, wrong ball in
-                                                                                // feeder
-        I(true, true, false, false, 0, INDEX_IN_SPEED, INDEX_IN_SPEED, true); // wrong ball in both
+        A(false, false, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, false, "no balls in system"), //
+        B(true, false, true, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, false, "Correct ball in ingest"), //
+        C(true, false, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, true, "Wrong ball in ingest"), //
+        D(false, true, false, true, INTAKE_IN_SPEED, INDEX_IN_SPEED, 0, false, "Correct ball in feeder"), //
+        E(false, true, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, true, "Wrong ball in feeder"), //
+        F(true, true, true, true, 0, 0, 0, false, "Correct ball in both"), //
+        G(true, true, false, true, INTAKE_OUT_SPEED, INDEX_OUT_SPEED, 0, false, "Wrong ingest, correct feeder"), //
+        H(true, true, true, false, 0, INDEX_IN_SPEED, INDEX_IN_SPEED, true, "Correct ingest, wrong feeder"), //
+        I(true, true, false, false, 0, INDEX_IN_SPEED, INDEX_IN_SPEED, true, "Wrong in both");//
 
         boolean ingestSensor, feederSensor, ingestColor, feederColor, shooterMisfire;
         double intakeMotorSpeed, ingestMotorSpeed, feederMotorSpeed;
+        String state;
```

Comment:
This (and the other fields in the `Bitmap` enum) should be private to avoid unwanted accesses.